File: argentina/Argentina.c
#include <stdio.h>

int main() {
    int pesos;
    scanf("%d", &pesos);
    int dollars;
    scanf("%d", &dollars);
    
    if (dollars * 50l > pesos) puts("Pesos");
    else puts("Dollars");
    
    return 0;
}

File: argentina/Argentina.java
import java.util.Scanner;

public class Argentina
{
    public static void main(String[] args) {
        final Scanner input = new Scanner(System.in);
        final int pesos = input.nextInt();
        final int dollars = input.nextInt();
        
        System.out.print(dollars * 50l > pesos ? "Pesos" : "Dollars");
    }
}
